stable/anchore-admission-controller/values.yaml

```diff
@@ -15,6 +15,11 @@ service:
   externalPort: 443
 
 apiService:
+  webhook:
+    enabled: true
```

Comment:
The expectation here is that to remove the configuration a user manually deletes it, or updates the values and redeploys? I can't recall if helm will remove the config if enabled = false on upgrade.

Reply:
behaviour is configurable - depends on what you want.
If you want the webhook to be retained, then https://helm.sh/docs/howto/charts_tips_and_tricks/#tell-helm-not-to-uninstall-a-resource can be used.